mgmtfn/k8splugin/kubeClient.go

```diff
@@ -284,9 +284,10 @@ func (c *APIClient) WatchServices(respCh chan SvcWatchResp) {
 				continue
 			}
 
-			//if wss.Object.ObjectMeta.Namespace != "default" {
-			//	continue
-			//}
+			if strings.Compare(wss.Object.ObjectMeta.Namespace, "kube-system") == 0 {
```

Comment:
Also, it might be better to ignore the kube-system namespace, instead of just processing the default namespace